datalake/src/main/java/com/sequenceiq/datalake/service/SdxEventsService.java

```diff
@@ -143,8 +144,8 @@
     }
 
     private List<CDPStructuredEvent> sortAndFilterBasedOnPageSize(List<CDPStructuredEvent> eventList, Integer size) {
-        return eventList.stream().sorted(Comparator.comparingLong(f -> f.getOperation().getTimestamp())).collect(toList())
-                .subList(0, (eventList.size() > size) ? size : eventList.size());
+        return eventList.stream().sorted(Collections.reverseOrder(Comparator.comparingLong(f -> f.getOperation().getTimestamp())))
+                .collect(toList()).subList(0, (eventList.size() > size) ? size : eventList.size());
```

Comment:
This change would reverse the order of the events. Currently, the events in the UI are listed in the correct order.

Before we proceed on this let's understand how the UI is rendering the events currently. Is it again performing a sort based on the timestamp?

Reply:
Yes I already tested this, the UI resorts them, otherwise they wouldn't be in the right order in the first place.

Reply:
how can the user show the first events?

Reply:
What do you mean by that? As in how does the user look at the very first events if there are more than the default max events?
This is currently not possible in the UI from what I've seen, only way is to send a request to the API with the very first page.

I think including the ability to see all events is nice, but outside the scope of this ticket.